### PR TITLE
Neubias workshop

### PR DESCRIPTION
**Pull request by jburel**
Add neubias to list of workshops

### CODE REVIEW

File: events/index.html [hunk 1]
```diff
@@ -11,14 +11,18 @@
             <div class="small-12 medium-8 medium-offset-2 columns">
                 <h6><i class="fa fa-calendar"></i> November 03, 2018</h6>
                 <h2><a href="https://docs.google.com/spreadsheets/d/1KbX0lecANaqspUf-l2YYWEp3W4VaoBa1p06Hci7HSss/edit#gid=2059970125">OMERO Workshop, Okinawa</a></h2>
-                <p class="card-caption">A 2h workshop on Data Management with OMERO.</p>
+                <p class="card-caption">Global Bio Imaging: a 2h workshop on Data Management with OMERO.</p>
+            </div>
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> October 18, 2018</h6>
+                <h2><a href="http://eubias.org/NEUBIAS/training-schools/staff/ts9-edinburgh-2018/">OMERO Workshop, Edinburgh</a></h2>
+                <p class="card-caption">NEUBIAS training school: focus on OMERO and Matlab integration.</p>
```
Comment (mtbc) [inline events/index.html]: Should it be "MATLAB"?

**Comment by will-moore** [comment]
Seems like both those gdocs are fully editable to anyone with the URLs? Maybe not great to make that them public?
Is there another way to make those details public (too much trouble to copy/paste them into our own page?). Still being updated?

**Comment by jburel** [comment]
@will-moore I have removed the g.doc for now I will add a link when it is more final
Probably next week after Basel

**Comment by jburel** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/

**Comment by will-moore** [comment]
Looks good, thanks

**Pull request merged**